terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -249,6 +276,30 @@ resource "aws_security_group_rule" "waf_lb_allow_web_users" {
   ipv6_cidr_blocks  = ["::/0"]
 }
 
+resource "aws_security_group_rule" "egress-to-prtg" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "egress"
+  description              = "allow web traffic to get to prtg"
```

Comment:
Can you correct these descriptions? This is allowing access to the portal server from the import machine

Reply:
= "allow portal server access from import server"